core/blockly.ts

```diff
@@ -26,7 +26,7 @@ import {BlockSvg} from './block_svg.js';
 import {BlocklyOptions} from './blockly_options.js';
 import {Blocks} from './blocks.js';
 import * as browserEvents from './browser_events.js';
-import {Bubble} from './bubble.js';
+import {Bubble} from './bubble_old.js';
```

Comment:
What is your plan for the `bubble_old` and `icon_old` classes? How long will those stick around?

Do you need to export the new bubble class here?

Reply:
They will stick around until all of the builtin bubbles + icons get converted to subclass the new classes.

So the old bubble will go away after #7045 , #7044, and #7043 are resolved.

And the old icon will go away after #7040, #7039, and #7038 are resolved.

Reply:
OK, makes sense. Can you add an issue then to check that we re-export the appropriate new classes from `blockly.ts` once everything is in final form? I don't want to miss that when we do the release.